### ISSUE TITLE
cannot build android after migration from capacitor-fcm@2.0.0

### ISSUE THREAD

**Issue by josh-m-sharpe** [capacitor-community/fcm#43]
Android production builds are failing with the following now that we've migrated to this version of the plugin.

```
* What went wrong:
Could not determine the dependencies of task ':app:collectReleaseDependencies'.
> Could not resolve all task dependencies for configuration ':app:releaseRuntimeClasspath'.
   > Could not resolve project :capacitor-community-fcm.
     Required by:
         project :app
      > Unable to find a matching configuration of project :capacitor-community-fcm:
          - None of the consumable configurations have attributes.
```

**Comment by josh-m-sharpe** [comment]
To clarify, we upgrade to @capacitor-community/fcm@1.0.4

**Comment by arraymonkey** [comment]
I had the same problem, try editing the capacitor.settings.gradle and add 'capacitor-fcm' to the src.
from: 
include ':capacitor-community-fcm'
project(':capacitor-community-fcm').projectDir = new File('../node_modules/@capacitor-community/fcm/android')

to:

include ':capacitor-community-fcm'
project(':capacitor-community-fcm').projectDir = new File('../node_modules/@capacitor-community/fcm/android/capacitor-fcm')

**Comment by stewwan** [comment]
fixed in 1.0.5
please check out the android breaking change on [changelog](https://github.com/capacitor-community/fcm/blob/master/CHANGELOG.md#breaking-change)
